@here/harp-theme-tool/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@here/harp-theme-tools",
+    "version": "0.12.0",
+    "description": "Tools for harp.gl-theme optimiziation.",
+    "main": "lib/index.js",
```

Comment:
Minor - we're not delivering any `index.js`  - remove.

Reply:
done